app/lib/clients/vault/identity.rb

```diff
@@ -0,0 +1,68 @@
+module Clients
+  class Vault
+    module Identity
+      def put_entity(name, policies)
+        write_identity(path: "identity/entity",
+                       name: name,
+                       policies: policies,
+                       extra_params: [ :metadata, :disabled ])
+      end
+
+      def put_group(name, policies)
+        write_identity(path: "identity/group",
+                       name: name,
+                       policies: policies,
+                       extra_params: [ :metadata, :type, :member_group_ids, :member_entity_ids ],
+                       defaults: { type: "external" })
+      end
+
+      def read_entity(sub)
```

Comment:
since we are using the "entity/name" endpoint, I would change the param name to "name"

Reply:
done